lib/errors/partialError.js

```diff
@@ -1,8 +1,8 @@
 const KuzzleError = require('./kuzzleError');
 
 class PartialError extends KuzzleError {
```

Comment:
you need to update this class' `toJSON` overload

Reply:
right thanks